CMakeLists.txt

```diff
@@ -62,6 +62,10 @@ if(MSVC)
     add_definitions("-D_CRT_SECURE_NO_WARNINGS -D_SCL_SECURE_NO_WARNINGS")
 endif()
 
+if(NOT CHFL_SYSTEM_LZMA)
+    add_definitions("-DLZMA_STATIC_API")
```

Comment:
Thanks, this fixes the build on Windows. I actually made a branch for this but never submitted the PR :roll_eyes: . I'll go delete that.

Reply:
I don't think is fixes the build, I still get errors on the static builder ... I'm trying to build it locally in a VM, but it is super slow =(

Reply:
I have a windows box I was working on.